extensions/applicationinsights-properties-js/src/Context/User.ts

```diff
@@ -2,9 +2,23 @@
 // Licensed under the MIT License.
 
 import { ITelemetryConfig } from '../Interfaces/ITelemetryConfig';
-import { Util, IUser, IUserContext } from '@microsoft/applicationinsights-common';
+import { Util, IUserContext } from '@microsoft/applicationinsights-common';
```

Comment:
Was this Util import missed?

Reply:
No, the class is still using the Util.removeStorage(), I have not **yet** refactored out all of the Util functions like I did for CoreUtils as the Cookie Manager change is also refactoring the Util set / get cookie functions, so I didn't want to keep adding onto this PR and complicate the merge than it already was 😄